src/cbc_sdk/platform/alerts.py

```diff
@@ -1794,6 +1796,51 @@ def set_group_by(self, field):
 
         return grouped_alert_search_query
 
+    def export(self):
+        """
+        Starts the process of exporting Alerts from the organization in CSV format.
+
+        This starts an asynchronous job.  Wait for completion of the job and
+        then use one of the get results methods on the Job.
+
```

Comment:
Only 1 blank line between sections in the docstring.  Also applies below the example.

Reply:
done